### PR TITLE
[OSDOCS-6250]: Update to hosted control planes DR docs

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSDOCS-6250

Link to docs preview: https://60942--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-backup-restore-dr.html#dr-hosted-cluster-within-aws-region-script_hcp-backup-restore-dr

QE review:
- [x] QE has approved this change.

Additional information: This PR adds a link to where the migration script for disaster recovery for hosted control planes is sourced. Eric Rich and Dave Mulford have taken a look and stated that the link is OK to use while hosted control planes are in Tech Preview.

### CODE REVIEW

**Review by jparrill — COMMENTED** [review]
/lgtm

**Comment by jiezhao16** [comment]
lgtm

**Review by opayne1 — COMMENTED** [review]
Had a small thought but otherwise lgtm!

File: modules/dr-hosted-cluster-within-aws-region-script.adoc [hunk 1]
```diff
@@ -63,4 +63,6 @@ EXTERNAL_DNS_DOMAIN="guest.jpdv.aws.kerbeross.com"
 source <env_file>
 ----
 +
-where: `env_file` is the name of the file where you saved the script.
\ No newline at end of file
+where: `env_file` is the name of the file where you saved the script.
+
+The migration script is maintained at the following repository: link:https://github.com/openshift/hypershift/blob/main/contrib/migration/migrate-hcp.sh[https://github.com/openshift/hypershift/blob/main/contrib/migration/migrate-hcp.sh].
```
[the remark this answers is not in the document]
Comment (lahinson) [reply modules/dr-hosted-cluster-within-aws-region-script.adoc:68]: Good thought. I can nest it under the code block so it looks less out of place.

**Review by jldohmann — APPROVED** [review]
LGTM

**Pull request merged**